cmd/manager/aggregator.go

```diff
@@ -542,6 +542,20 @@ func createResults(crClient aggregatorCrClient, scan *compv1alpha1.ComplianceSca
 		return nil
 	}
 
+	// This feels like an appropriate seam to implementing forwarding since
+	// we have a list of consistent results and remediations. Determine if
+	// we're configured to forward results to an external system and
+	// establish that connection here.
+	//
+	// f := getForwardingClient()
+	//
+	// The getForwardingClient() function is a factory that returns the
```

Comment:
I think a factory based on settings is a good way to go.

Reply:
Yeah - I like that idea too since I can think of at least three forwarding implementations.

1.) gRPC forwarding
2.) log forwarding (for testing)
3.) noop forwarding (returned by default when no forwarding is configured)

I'd like to keep all the logic to determine which implementation to use isolated to the factory and not let that bleed into the aggregator code.